### PR TITLE
Added new Dropship unit

### PR DESCRIPTION
**Pull request by Dzierzan**
Basic implementation of Dropship unit. It requires further balance and polishing and it's not in the scope of this pull request.

unknown

### CODE REVIEW

File: mods/hv/rules/aircraft.yaml [hunk 1]
```diff
@@ -272,3 +272,59 @@ MOTHER2B:
...
+	Inherits: ^Helicopter
+	Valued:
+		Cost: 900
+	Tooltip:
+		Name: Heavy Transport Dropship
+	Buildable:
+		Queue: Aircraft
+		BuildPaletteOrder: 50
+		# YI needs a counterpart transport plane.
+		Prerequisites: radar
```
Comment (Mailaender) [inline mods/hv/rules/aircraft.yaml:286]:
```suggestion
		Prerequisites: radar, ~aircraft.yi
```

File: mods/hv/weapons/missiles.yaml [hunk 2]
```diff
@@ -286,3 +286,23 @@ Patriot:
 		Explosions: small
 		ImpactSounds: explosion08.wav
 		ImpactActors: false
+
+DropshipMissile:
+	Inherits: ^AntiGroundMissile
+	ReloadDelay: 100
+	Range: 9c0
```
Comment (Mailaender) [inline mods/hv/weapons/missiles.yaml:293]: > RangeLimit lower than weapon range!

File: mods/hv/rules/aircraft.yaml (hunk 1 above)
Comment (GraionDilach) [reply mods/hv/rules/aircraft.yaml:286]: This was copied from #223.

**Comment by Dzierzan** [comment]
Like I stated. The whole purpose of this PR was to provide art and basic implementation. Matze or Graion can work on it from here.

**Pull request merged**